salt/grains/core.py

```diff
@@ -1156,6 +1156,23 @@ def os_data():
         grains.update(_windows_cpudata())
         grains.update(_windows_virtual(grains))
         grains.update(_ps(grains))
+
+        if 'Server' in grains['osrelease']:
```

Comment:
How do you know that non-server Windows systems don't? I'm not sure I understand what prevents that from happening.